src/Parlot/Fluent/ParseContext.UntypedScope.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Parlot.Fluent
+{
+    public partial class ParseContext
```

Comment:
I don't understand why it should be part of the library. This PR allows to change the type of `ParseContext`, so any project could then create their own, and have one that takes a dictionary if they need, or custom properties.

Reply:
indeed, I just thought having an example and the possiblity to use such a parser straight ahead would be the right thing to do. but I agree with you